codemod/spotlight_util.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+
+# Copyright (c) 2007-2008 Facebook
+
+from Cocoa import *
```

Comment:
Yes, you can write the following:

```py
try:
    import Cocoa
except ImportError:
    print("lol nah")
```

As for my Python install:

```
$ which python
/usr/local/bin/python
$ python --version
Python 2.7.10
```

I don't think Python on OS X typically has access to `import Cocoa`. It appears to be defined as part of https://bitbucket.org/ronaldoussoren/pyobjc/, so users will have to install that module. I assume you must have run `pip install pyobjc` at some point...?

To ensure users have this new dependency, you'll have to add a `requires=['pyobjc']` to this project's `setup.py`, and including it such that it's only a dependency on OS X host environments. Personally I'm not gung-ho on including this dependency, but I'm also not vehemently against it.

Perhaps conditionally using it if it's available is the answer? I wonder if `setup.py` supports the concept of optional dependencies?